tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist = True
-envlist = py37, py38, py39, py310, pep8, black
+envlist = py37, py38, py39, py310, py311, pep8, black
```

Comment:
I think we also need to update the section `[gh-actions]` below.

Reply:
Right, and the Actions workflow.